### PR TITLE
Suzuki-survey(Tara)

### PR DESCRIPTION
**Pull request by trangtara**
Link to my deployed project:

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

**Review by idsintehittapa — APPROVED** [review]

**Review by idsintehittapa — CHANGES_REQUESTED** [review]
Hey Tara!

Nice work with your survey! I learned a lot going through your code 👍

You have fulfilled the blue level requirements

at least 3 questions
radio and select as form inputs
you have the submit button that gives a summary of the answers to the user
the site follows accessibility guidelines (here I learned a lot from you!)
Also, some red requirements

I think, you used a form element you haven't tried before? (it's a bit unclear if you did, maybe this could be specified in the readme file or when you did the PR)
You added some validation to the form
And lastly, one black level requirement too!

one question at the time.
You have some validation on your form, I remember that you had some problems with this from the QA on Friday. I am guessing the window.location.hash makes it difficult to validate since you can basically just scroll pass the questions (even the required ones). To add some more validation you could have added a disable on the last submit button so the user cannot come to the summary - see my comment.

For this weeks project, try to remember to also include a general comments that to help the peer review when looking through your code.

Great job!

File: code/src/UserRating.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    label:"Good",
+    id:"good"
+  },
+  {
+    value:"excellent",
+    label:"Excellent",
+    id:"excellent"
+  }
+  // 'Average',
+  // 'Good',
```
Comment (EmOlsz) [inline code/src/UserRating.js:30]: Commented code should be deleted as well

**Review by EmOlsz — COMMENTED** [review]
Great job with outer packages for HTML elements and for custom scroll Tara! Very impressive :)
As Johanna pointed out, fix semicolons occurrence and delete `br` tags, after that we will be able to approve whole project

File: code/src/SurveyForm.js [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    <div>
+      {!showSummary &&
+        <form id="surveyForm" className="survey-container" onSubmit={event => event.preventDefault()}>
+          <SnapSection>
+            <UserCarModels
+              carModels={carModels}
+              onCarsChange={handleCarsChange}
+            />
+            <NextButton
+              sectionId="sectionReasonsToBuy"
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/SurveyForm.js:82]: I will work on this suggestion later as I am not sure how to connect the button validation with the scroll snap property.

File: code/src/SurveyForm.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+              disabled={!email.includes('@')}
+            />
+          </SnapSection>
+          <SnapSection>
+            <TextMessage
+              textMessage={textMessage}
+              onMessageChange={handleMessageChange}
+            />
+            <SubmitButton
+              doSubmit={handleSubmit}
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/SurveyForm.js:134]: I will work on this validation later after I figure out how to connect it with the scroll snap property.

File: code/src/UserCarModels.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+const selectCarModels = [
+ { value: "suzuki Ertiga", label: "Suzuki Ertiga"},
+ { value: "suzuki swift", label: "Suzuki Swift" },
+ { value: "suzuki XL7", label: "Suzuki XL7"},
+ { value: "suzuki ciaz", label: "Suzuki Ciaz"},
+ { value: "super carry pro", label: "Super Carry Pro"},
+ { value: "carry truck", label: "Carry Truck"},
+ { value: "carry van", label: "Carry Van"}
+];
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/UserCarModels.js:12]: I found this on google :-), not a thing that I should take credit for :-)

File: code/src/UserEmail.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  return (
+    <div className="section-user-email" id="sectionUserEmail">
+      <label htmlFor="userEmail">Your Email (required)<sup aria-hidden="true">*</sup></label>
+      <input 
+        type="email" 
+        id="userEmail"
+        className="email-input"
+        onChange={(event) => onEmailChange(event.target.value)}
+        value={email}
+        required
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/UserEmail.js:13]: Definitely, the validation can be improved.

File: code/src/index.css [hunk 6]
```diff
@@ -1,13 +1,226 @@
...
+  border: none;
+  outline: none;
+  background-color: transparent;
+  font-size: 30px;
+  line-height: 38px;
+  overflow-wrap: break-word;
+  resize: none;
+  overflow: hidden;
+  transition: box-shadow 0.1s ease-out 0s;
+  box-shadow:  rgba(92, 78, 78, 0.3) 0px 2px;
```
[the remark this answers is not in the document]
Comment (trangtara) [reply code/src/index.css]: Thanks for the recommendation. I will use it for the next projects.

File: code/src/UserRating.js (hunk 1 above)
Comment (trangtara) [reply code/src/UserRating.js]: Deleted commented codes

**Review by trangtara — COMMENTED** [review]
added semicolons after declaring functions and objects.
deleted <br> between label and input, replaced with css styling.
replace 0px with 0.
deleted improper double space and commented code.
The improvement point is to figure out how to combine input validation with the scroll snap property.

**Pull request closed without merging**